limap/line2d/dense/dense_matcher/roma.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import romatch
```

Comment:
romatch is not included in requirements.txt or dependencies

Reply:
How about we making a separate installation, as we now already make HAWP, LBD separately installed to reduce dependencies.